deb-package/pkg/lib/systemd/system/unikernel.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Unikernel Service
+After=network.target
+Before=dvoting.service
+
+[Service]
+User=dvoting
+
+StandardInput=tty
+StandardOutput=tty
+TTYPath=/dev/tty10
+
+ExecStartPre=/bin/bash -c "/opt/dedis/dvoting/bin/create-bridge /etc/dedis/dvoting/config.env"
+ExecStart=/bin/bash -c "/opt/dedis/dvoting/bin/start-unikernel /etc/dedis/dvoting/config.env"
```

Comment:
This makes me uncomfortable: it seems very brittle to me.

1) the deployment paths are now read from a configuration file, which means we can't update them later on.
We've just prevented ourselves from moving things with an update
At the very least, I would move the paths in a directory other than /etc/ , which FPM will associate with a configuration file.
Even better would be to hardcode the paths and not pass them as parameters.

2) normally, shell scripts can be run directly by systemd (e.g. https://tecadmin.net/run-shell-script-as-systemd-service/ ). Why do we need to run them in a bash subprocess ?